airgun/views/contentview_new.py

```diff
@@ -56,3 +88,148 @@ def is_displayed(self):
     def after_fill(self, value):
         """Ensure 'Create content view' button is enabled after filling out the required fields"""
         self.submit.wait_displayed()
+
+
+class ContentViewEditView(BaseLoggedInView):
```

Comment:
Please add doc string for a class which would help to understand added functionality.